lms/djangoapps/courseware/migrations/0012_auto__add_queriestemporary__add_queriessaved__add_groupedtempqueriessu.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we actually move this model and its migrations to a newly created django app.  I've learned the lesson -- we should not have multiple (0012) migrations in the same app, which could happen if we don't merge this upstream.